src/app/Pages/Blog.tsx

```diff
@@ -0,0 +1,105 @@
+          </CardTitle>
+          <CardBody isFilled={false}>
+            <Bullseye>
+              <img src={post11png}
+              style={{
+                display: 'flex',
+                justifyContent: 'center',
+              }}></img>
+            </Bullseye>
+            <TextContent>
+              <Text component={TextVariants.p}>
+                Our Cloud Experience team is excited to announce the availability of the first version of our Cloud Image Directory.
+                This tool is in an experimental stage, however we have decided to release this early access version to our community
+                and stay true to our open culture and engineering principles and invite you to join us in this journey.
+              </Text>
+
+              <Text component={TextVariants.p}>
+                Based on community feedback, we realized that many users today struggle to find out when and where new images are available,
+                because they are neither actively promoted on cloud marketplaces, nor easily accessible on other available platforms.
+                In addition, research suggests that experienced users will benefit from searching for particular versions of cloud images,
+                like specific older versions. Finally, you can use this tool to try out your preferred images before considering other requirements,
+                such as specific subscriptions and licenses.
+              </Text>
+
+              <Text component={TextVariants.h3}>
+                What this new tool is about
+              </Text>
+
+              <Text component={TextVariants.p}>
+                Our  new tool provides easy access to all Linux cloud images.  Our vision is to help users search and quickly find any publicly
+                available cloud image so they can conveniently deploy it to their preferred environment.
+                It provides an easy one-step dashboard that points you to the latest cloud image release on the marketplace of your choice.
+                If a single-click spin-up for the newest version of your favorite distribution is what you are looking for, you’re already set.
+                If you are looking for a cloud image that comes with a specific version or feature, use the provider-specific browser to search,
+                find, and select from a variety of offers.
+              </Text>
+
+              <Text component={TextVariants.h3}>
+                How to use it
+              </Text>
+
+              <Text component={TextVariants.p}>
+                You can access the Cloud Image Directory by navigating to https://imagedirectory.cloud/. On the landing page, you can select any of
+                the available cloud providers, which redirects you to the latest RHEL Image version available in the respective marketplace.
+                In addition, you can use our Red Hat image builder to customize your own image by selecting “Build now”.
```

Comment:
Should we add a link to image builder?

Reply:
Yes, let's do that. Builds more of that idea of choice.